doc_src/cmds/status.rst

```diff
@@ -146,6 +147,32 @@ The following operations (subcommands) are available:
     Currently the only available *FEATURE* is :ref:`scroll-content-up <term-compat-indn>`.
     An error will be printed when passed an unrecognized feature.
 
+.. _status-language:
+
+**language**
+    Show or modify message localization settings.
+    When invoked without arguments, the current language settings are shown.
+
+    Available subcommands:
+
+    **list-available**:
+    Prints the language names for which fish has translations.
+    These names can be used with the **set** subcommand.
+
+    **set**:
+    Sets the language precedence for fish's messages.
```

Comment:
I probably wouldn't capitalize the sentence after colon,
since it's not a full sentence (https://www.grammar-monster.com/punctuation/colons_capitalization_after_a_colon.html)